### PR TITLE
Chore Create Default Users

### PR DESCRIPTION
**Pull request by Cheza-Dzabala**
#### What does this PR do?
Creates seeders for default Users and Posts

#### Description of Task to be completed?
Create default seeders so that tests can run and sign in on existing accounts

#### How should this be manually tested?

1. Clone the repository
2. Run the ` npm install ` command
3. Run the ` npm run test ` command
4. Manually check your Postgres __TEST__ database for the following:  

- seeded user accounts: 

a. John Doe (johndoe@gmail.com)
b. Jane Doe (janedoe@gmail.com)

- seeded posts:

a. Description: 
> This is a demo post one. This post does the awesome post things. No post is more awesomer than this.

b. Description: 
> This is a demo post two! This post does the awesome post things. No post is more awesomest than this.

#### Any background context you want to provide?

N /A

#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

**Comment by alainmateso** [comment]
Please add the migration to add the image column to the posts table!

File: __test__/__posts__/getPostsTest.spec.js [hunk 1]
```diff
@@ -9,30 +9,14 @@ chai.use(chaiHttp);
...
 describe('Get Posts', () => {
-  it('Should not get any post', (done) => {
-    chai.request(app)
-      .get('/api/v1/posts')
-      .end((err, res) => {
-        expect(res.status).to.be.equal(404);
-        done();
-      });
-  });
+  // it('Should not get any post', (done) => {
```
Comment (sabin18) [inline __test__/__posts__/getPostsTest.spec.js]: you should remove the commented code here .

File: __test__/__posts__/getPostsTest.spec.js [hunk 2]
```diff
@@ -9,30 +9,14 @@ chai.use(chaiHttp);
...
+  //   chai.request(app)
+  //     .get('/api/v1/posts')
```
Comment (ngireric123) [inline __test__/__posts__/getPostsTest.spec.js]: remove these comments

**Review by alainmateso — APPROVED** [review]
It looks great!

**Review by ngireric123 — COMMENTED** [review]
it is ok now

**Review by ngireric123 — APPROVED** [review]

**Pull request merged**